src/catchmentmodel/LSDCatchmentModel.cpp

```diff
@@ -2345,7 +2345,7 @@ void LSDCatchmentModel::flow_route()
                 (elev[x][y] + water_depth[x][y])) / DX;
 
             if (x == imax) tempslope = edgeslope;
-            if (x <= 2) tempslope = 0 - edgeslope;
```

Comment:
Why is tempslope negative at the Western domain boundary, and positive at the Eastern domain boundary. (And similarly for N-S)?

This seems odd...